src/scss/register-layout.scss

```diff
@@ -1,47 +1,57 @@
 .layout {
     &.enterprise2 {
-
```

Comment:
This can be moved to `pages.scss`, because it provides styling for a specific page (the register/new account page). Just put a comment before this block identifying the page it relates to, because it's not super obvious from the code itself.